soroban-spec/fixtures/ts/src/convert.ts

```diff
@@ -130,3 +130,17 @@ export function addressToScVal(addr: string): xdr.ScVal {
     let addrObj = Address.fromString(addr);
     return addrObj.toScVal();
 }
+
+export function i128ToScVal(i: bigint): xdr.ScVal {
+    return xdr.ScVal.scvI128(new xdr.Int128Parts({
+        lo: xdr.Uint64.fromString((Number(i & BigInt(0xFFFFFFFFFFFFFFFFn))).toString()),
```

Comment:
Can we use `Number` for a 64-bit value? My understanding was no, because Number is max 53-bit. cc @paulbellamy @Shaptic

Reply:
Good point, I'll change it.

Reply:
No you cannot, but pls keep an eye out for when https://github.com/stellar/js-stellar-base/pull/620 lands! (Soon:tm:)